services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -121,6 +121,21 @@ def storage_endpoint(self) -> str:
         return f"{self.STORAGE_HOST}:{self.STORAGE_PORT}"
 
 
+class CatalogSettings(BaseCustomSettings):
+    CATALOG_HOST: str = "catalog"
+    CATALOG_PORT: int = 8000
+    CATALOG_VTAG: str = "v0"
+
+    @cached_property
+    def endpoint(self) -> str:
+        return AnyHttpUrl.build(
+            scheme="http",
+            host=self.CATALOG_HOST,
+            port=f"{self.CATALOG_PORT}",
+            path=f"/{self.CATALOG_VTAG}",
+        )
```

Comment:
I will refactor this when merging #2988
It was moved to settings-library

Reply:
good